docs/enterprise/installing.md

```diff
@@ -1,6 +1,6 @@
 # 安装
 
-fisco generator依赖python,openssl以及curl。使用前请检查是否满足依赖
+fisco generator依赖python,openssl以及curl,nc工具。使用前请检查是否满足依赖，同时，需要先满足fisco bcos启动时的条件，参考[依赖安装]()https://fisco-bcos-documentation.readthedocs.io/zh_CN/feature-2.0.0/docs/manual/install.html?highlight=%E4%BE%9D%E8%B5%96#id4

```

Comment:
Make sure the grammar of the markdown is right.

Reply:
modify it now